benchmarks/bytes_and_flops/main.cpp

```diff
@@ -44,15 +44,17 @@
 
 #include <Kokkos_Core.hpp>
 #include <Kokkos_Timer.hpp>
-#include <bench.hpp>
 #include <cstdlib>
 
+template <class T>
+void run_stride_unroll(int, int, int, int, int, int, int, int);
```

Comment:
No.  Keep the header include and do `extern template ...` to skip the instantiation from that compile unit.
(I pushed a fix directly to your branch)